gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -246,6 +246,9 @@ public void onCommandsSelected(CommandIndex index) {
 
   @Override
   public void onProfileLoaded(Loadable.Message error) {
+    if (error != null) {
+      tree.profileLoadingError = error;
+    }
```

Comment:
Here, once tree.profileLoadingError is set to an error, there is no way to turn it back to null. Would it make sense to just have:

```suggestion
    tree.profileLoadingError = error;
```

such that one can retry to profile and, if the second attempt to profile works, get rid of the error?

Reply:
Nice catch! Done with the new commit!